nixos/modules/services/mail/mailman.nix

```diff
@@ -0,0 +1,120 @@
+{ config, pkgs, lib, ... }:          # mailman.nix
+
+with lib;
+
+let
+
+  cfg = config.services.mailman;
+
+  pythonEnv = pkgs.python37.withPackages (ps: [ps.mailman]);
```

Comment:
I suggest using `python3` instead, because only the current default python 3 version gets cached versions, 3.7 won't be default forever. Same for `python37Packages` below

Reply:
Fixed.